mindmeld/components/question_answerer.py

```diff
@@ -100,9 +121,11 @@ def get(self, index, size=10, query_type="keyword", **kwargs):
         Returns:
             list: A list of matching documents.
         """
-
         doc_id = kwargs.get("id")
 
+        if not query_type:
+            query_type = self._query_type
```

Comment:
We can follow this pattern for concision :)

```suggestion
            query_type = query_type or self._query_type
```

Reply:
fixed